clowdlet/models/clowdapp.py

```diff
@@ -0,0 +1,48 @@
+"""ClowdApp model module."""
+from pathlib import Path
+from typing import Any
+
+from clowdlet.models.deployment import Deployment
+from clowdlet.utils import replace_vars
+
+
+class ClowdApp:
+    """Class representing single ClowdApp containing multiple deployments and other objects."""
+
+    def __init__(self, clowdapp_yaml: dict[str, Any], parameters: dict[str, str]):
+        self.name: str = replace_vars(clowdapp_yaml.get("metadata", {}).get("name", "clowdapp"), parameters)
+        self.deployments: list[Deployment] = []
+
+        self._parse_deployments(clowdapp_yaml.get("spec", {}).get("deployments", []), parameters)
+
+    def _parse_deployments(self, deployments: list[dict[str, Any]], parameters: dict[str, str]) -> None:
+        for deployment in deployments:
+            self.deployments.append(Deployment(deployment, parameters, self.name))
+
+    def render_quadlet_network(self) -> str:
+        """Return string representing quadlet unit compatible with systemd."""
+        lines = [
+            "[Unit]",
+            "After=network-online.target",
+            "",
+            "[Network]",
+            f"NetworkName={self.name}",
+            "",
+            "[Install]",
+            "WantedBy=multi-user.target default.target",
+        ]
+        return f"{"\n".join(lines)}\n"
```

Comment:
linter fails because in python versions lower than 3.12 it is not possible to use double quotes inside a f-string if f-string uses double quotes

this should fix the issue IMO
```suggestion
        return f"{'\n'.join(lines)}\n"
```

Reply:
changed